src/TrendsOverview.svelte

```diff
@@ -288,7 +288,7 @@
                 ? "map-trend-selector-button map-trend-selector-selected"
                 : "map-trend-selector-button"}
               on:click={onChangeMapTrend}
-              title="Search interest in any aspect of COVID-19 vaccination. A scaled value that you can compare across regions and times."
+              title="Search interest in any aspect of COVID-19 vaccination. For example, “when can i get the covid vaccine” or “cdc vaccine tracker”. A scaled value that you can compare across regions and times. This parent category includes searches from the other two subcategories."
```

Comment:
I notice the "smart quotes" here instead of plain quotes.  Looks like these UTF characters will be properly encoded, but we should make a mental note for consistency's sake that any quotes in the body text should use fancy quotes and not normal quotes.